menu/models.py

```diff
@@ -85,18 +85,20 @@ class MenuItem(models.Model):
         help_text=_(u'Should this item only be shown to non-logged-in users?')
         )
 
-    is_dynamic = models.BooleanField(
-        _(u'Dynamic mode'),
+    is_reverse_url = models.BooleanField(
+        _(u'Is reverse URL'),
         blank=True,
         default=False,
-        help_text=_(u'Should this is a dynamic URL?')
+        help_text=_(u'Should Link URL need reverse?')
```

Comment:
Suggested help text: "Use URL resolver for this link? If False, this is treated as a literal URL.